spec/requests/api/querying_spec.rb

```diff
@@ -103,17 +103,17 @@ def create_vms_by_name(names)
     end
 
     it 'supports sql friendly virtual attributes' do
-      host_foo =  FactoryGirl.create(:host, :name => 'foo')
       host_bar =  FactoryGirl.create(:host, :name => 'bar')
-      vm_foo = FactoryGirl.create(:vm, :name => 'vm_foo')
-      vm_bar = FactoryGirl.create(:vm, :name => 'vm_bar')
-      host_foo.vms << vm_foo
-      host_bar.vms << vm_bar
+      host_zap =  FactoryGirl.create(:host, :name => 'zap')
+      host_foo =  FactoryGirl.create(:host, :name => 'foo')
+      FactoryGirl.create(:vm, :name => 'vm_bar', :host => host_bar)
+      FactoryGirl.create(:vm, :name => 'vm_foo', :host => host_foo)
```

Comment:
This looks good, but I might recommend that you create the foo one first (since you expect it to be in the middle) to prove that it ordered them by this attribute